lego/apps/users/fixtures/test_abakus_groups.yaml

```diff
@@ -545,3 +545,22 @@
     rght: 2
     tree_id: 19
     level: 0
+
+- model: users.abakusgroup
+  pk: 29
+  fields:
+    deleted: false
+    name: PR
+    description: ''
+    contact_email: ''
+    parent: 21
+    logo: null
+    type: komite
+    text: ''
+    permissions: '["/sudo/admin/companyinterest/", "/sudo/admin/surveys/", "/sudo/admin/submissions/"]'
+    show_badge: true
+    active: true
+    lft: 5
+    rght: 6
+    tree_id: 18
+    level: 2
```

Comment:
It's not entirely obvious (we should add a header to this file), but this file is auto-generated from `test_abakus_groups.py`. So this will be overwritten the next time someone runs the generate command.

Reply:
Okay, then I will just add PR to test_abakus_groups.py